### ISSUE TITLE
yarn content move errors

### ISSUE THREAD

**Issue by Elchi3** [mdn/yari#2971]
On latest yari, do a move like

`yarn content move Web/API/DocumentOrShadowRoot/fullscreenElement Web/API/Document/fullscreenElement`

Output:

```
Will move 1 documents from Web/API/DocumentOrShadowRoot/fullscreenElement to Web/API/Document/fullscreenElement for en-US
Web/API/DocumentOrShadowRoot/fullscreenElement → Web/API/Document/fullscreenElement
? Proceed? Yes
Trying to resolve a non-en-us path for /zh-CN/docs/Web without CONTENT_TRANSLATED_ROOT set.
Trying to resolve a non-en-us path for /zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/create without CONTENT_TRANSLATED_ROOT set.
Trying to resolve a non-en-us path for /zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/seal without CONTENT_TRANSLATED_ROOT set.
Trying to resolve a non-en-us path for /zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/isSealed without CONTENT_TRANSLATED_ROOT set.
Trying to resolve a non-en-us path for /zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/preventExtensions without CONTENT_TRANSLATED_ROOT set.
Trying to resolve a non-en-us path for /zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze without CONTENT_TRANSLATED_ROOT set.
Trying to resolve a non-en-us path for /zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/isFrozen without CONTENT_TRANSLATED_ROOT set.

error: To-URL has to resolve to a file (/en-US/docs/Web/API/DocumentOrShadowRoot/fullscreenElement)

error Command failed with exit code 1.
```

Questions:

1. Are there docs on what to do regarding CONTENT_TRANSLATED_ROOT? Do I need to set this even though I don't want to do anything with translations?
2. `error: To-URL has to resolve to a file (/en-US/docs/Web/API/DocumentOrShadowRoot/fullscreenElement)` makes no sense to me. DocumentOrShadowRoot/fullscreenElement is the file that was moved and so, yes, that file doesn't exit anymore.

**Comment by Elchi3** [comment]
I can't really move pages anymore since this `yarn content move` aborts with the above errors and no `_redirect.txt` file updates happen.

**Comment by peterbe** [comment]
It happens because the tool that validates redirects gets invoked as part of the `move` CLI. This was a recent addition and the problem persists within the `yari` repo too (ie. `yarn tool move ...`). 

To get yourself unstuck, since @fiji-flo is on PTO today, consider cloning the whole `mdn/translated-content` repo somewhere and run:

```
export CONTENT_TRANSLATED_ROOT=~/dev/MOZILLA/MDN/translated-content/files
yarn content move ...
```

**Comment by Elchi3** [comment]
Thanks Peter! That fixes the first problem. I guess the real one is the second TO-URL error.

**Comment by peterbe** [comment]
What I find annoying is that it did things even if it failed. I'm trying to wrap my head around the code.

**Comment by peterbe** [comment]
I think I know what's going on. If you open the `_redirects.txt` file you'll find entries like this one:
```
/en-US/docs/Web/API/document.mozFullScreenElement       /en-US/docs/Web/API/DocumentOrShadowRoot/fullscreenElement
```
I.e. before Florian S came along, `/en-US/docs/Web/API/DocumentOrShadowRoot/fullscreenElement` was a prefectly good destination. 
So the error, which does seem confusing, actually makes sense. It's not choking on the new entry but on an existing entry. 
So what needs to happen is that all *old* entries where the destintion *was* `/en-US/docs/Web/API/DocumentOrShadowRoot/fullscreenElement` *also* needs to change.

**Comment by fiji-flo** [comment]
@Elchi3 could you take another look? This should be fixed by #2982

**Comment by Elchi3** [comment]
Thanks @fiji-flo, this works now as discussed in #2982.

Great work, I really appreciate you fixing this :)
